### PR TITLE
Awesome books: style core elements

### PR DESCRIPTION
**Pull request by ad9311**
### Greetings 👋🏽 

In this PR we present the changes suggested on: [Awesome books: style core elements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m3_style_core_elements.md) and we also added some colors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,33 +3,39 @@
...
+            <li>
+              <label class="form-label" for="author">Author:</label>
+              <input class="form-control" id="author" type="text">
+            </li>
+            <li class="pt-2">
+              <label class="form-label" for="title">Title:</label>
+              <input class="form-control" id="title" type="text">
+            </li>
+            <li>
+              <input class="btn btn-primary my-2" type="button" value="Add" onclick="addBooks()">
```
Comment (ZawadiSumba66) [inline index.html:34]: - [ ] As from the wireframe given while following the layout, the `add` button ought to be aligned to the right. Check this screenshot below:
image

File: index.html [hunk 2]
```diff
@@ -3,33 +3,39 @@
...
+        <h2 class="text-center">Add a new book</h2>
+        <form>
+          <ul>
+            <li>
+              <label class="form-label" for="author">Author:</label>
+              <input class="form-control" id="author" type="text">
+            </li>
+            <li class="pt-2">
+              <label class="form-label" for="title">Title:</label>
+              <input class="form-control" id="title" type="text">
```
Comment (ZawadiSumba66) [inline index.html]: - [ ] When you look carefully at the author and the title fields you will notice that each has a placeholder inside and there are no labels present. Please refer to the screenshot below.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
Hi Team 😄 , 

Great job styling your awesome books!! 👏 
However, there are some issues that you still need to work on to go to the next project but you are almost there! 💪 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ad9311** [comment]
The following changes have been made:

- The add button is in the right side of the form
- Labels have been removed, instead there are placeholders
- Title and author fields have been switched
- Add hr tag between h2 title and form

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @ad9311 and @Chike1990 , 

Great work with making changes requested by the previous TSE.
The project looks like the wireframe now :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**